src/implementations/vanilla/components/file/_file.scss

```diff
@@ -62,6 +62,7 @@ $_image-width: null !default;
   flex-basis: 50%;
   flex-grow: 1;
   flex-wrap: wrap;
+  flex-direction: column;
```

Comment:
with this change, the file type and size go under the name (instead of inline before)
Is that something requested by designers?
![image](https://user-images.githubusercontent.com/25579221/204546193-03002fe5-d2bf-4cac-ae14-b09c016f2338.png)

Reply:
uff, no it's not..thanks for noticing this.. ;)